Classes/Service/CacheService.php

```diff
@@ -71,6 +80,16 @@ public function flush($force = false)
     }
 
     /**
+     * Flushes caches using the data handler. This should not be necessary any more in the future.
+     * @deprecated
+     * @internal
+     */
+    public function flushCachesWithDataHandler()
+    {
+        $this->dataHandler->clear_cacheCmd('all');
```

Comment:
I thought about that. However clearing system caches is done beforehand already as the data handler in that case only uses the caching framework.

What other reasons do you have in mind calling this api with "system"?